editor/src/core/performance/__snapshots__/performance-regression-tests.spec.tsx.snap

```diff
@@ -2773,8 +2755,6 @@ Array [
   "/UtopiaSpiedFunctionComponent(View)/div/Symbol(react.forward_ref)(SpyWrapper)/UtopiaSpiedFunctionComponent(ComponentRenderer(App))",
   "/div/Symbol(react.forward_ref)(SpyWrapper)/UtopiaSpiedFunctionComponent(ComponentRenderer(App))/Symbol(react.forward_ref)(SpyWrapper)",
   "/div/Symbol(react.forward_ref)(SpyWrapper)/UtopiaSpiedFunctionComponent(ComponentRenderer(App))/Symbol(react.forward_ref)(SpyWrapper)",
-  "/div/Symbol(react.forward_ref)(SpyWrapper)/UtopiaSpiedFunctionComponent(ComponentRenderer(App))/Symbol(react.forward_ref)(SpyWrapper)",
```

Comment:
Sean do you know what change made the SpyWrapper re-render less?

Reply:
This comes from the overly eager rendering from this which is now a null value in its new location, as it should always update on that first render anyway. Restoring it, makes these renders return to the regression test.

Reply:
Context: https://github.com/concrete-utopia/utopia/pull/5120/files#diff-9be29e1233edaec82e4b10de584ebe2839adf2e913bdb4e8a53a79bdb9bb6b23R228

Reply:
okay!